README.md

```diff
@@ -12,7 +12,6 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
   - [Presentation](#presentation)
```

Comment:
- [ ] Kindly remove this link from the table of contents as it leads nowhere and as such is a broken link

Reply:
Thanks for your feedback. I will fix this 👍